templates/Dataland/v2/detail-cards.html.twig

```diff
@@ -9,12 +11,18 @@
         <div class="card-body">
             <p class="card-text text-dark">
                 <u>Funded By:</u> <br>
-                <span>{{ dataset.researchGroup.fundingCycle.fundingOrganization.name }}</span>
-            </p>
-            <p class="card-text text-dark">
-                <u>Funding Cycle:</u> <br>
-                <span>{{ dataset.researchGroup.fundingCycle.name }}</span>
+                {% set funders = [] %}
+                {% for funder in dataset.funders|default %}
+                    {% set funders = funders|merge([funder.name|trim('"')]) %}
+                {% endfor %}
+                <span>{{ funders|join('| ') }}</span>
```

Comment:
These are pipe delimited, the ticket says new line. Instead of doing the join, you can just push them out in the foreach as a `span`.